apps/test-suite/tests/FileSystemNext.ts

```diff
@@ -0,0 +1,95 @@
+'use strict';
+import * as FS from 'expo-file-system';
+import { File, Directory } from 'expo-file-system/next';
+import { Platform } from 'react-native';
+
+export const name = 'FileSystem@next';
+
+export async function test({ describe, expect, it, ...t }) {
+  const testDirectory = FS.documentDirectory + 'tests/';
+  t.beforeEach(async () => {
+    try {
+      await FS.makeDirectoryAsync(testDirectory);
+    } catch {}
+  });
+  t.afterEach(async () => {
+    try {
+      await FS.deleteAsync(testDirectory);
+    } catch {}
+  });
+
+  describe('FileSystem (Next)', () => {
+    it('Creates a lazy file reference', async () => {
+      const file = new File('file:///path/to/file');
+      // The path is normalized by the OS, with both being valid urls (https://stackoverflow.com/a/44725349)
```

Comment:
> I'm assuming the authors of Java aren't **incompetent**, so is there an explanation for why that's acceptable? ~ from the link you posted 

I found that funny 😆
However, I think we should ensure that the path format is the same on both platforms. I don't have a strong opinion about which one should be picked. If that stays as it is, that will confuse people.

Reply:
I think we should go with `file:///` as this is already used elsewhere.